packages/reactive-element/src/reactive-element.ts

```diff
@@ -893,25 +895,11 @@ export abstract class ReactiveElement
 
   constructor() {
     super();
-    this._initialize();
-  }
-
-  /**
-   * Internal only override point for customizing work done when elements
-   * are constructed.
-   *
-   * @internal
-   */
-  _initialize() {
-    this.__updatePromise = new Promise<boolean>(
-      (res) => (this.enableUpdating = res)
-    );
-    this._$changedProperties = new Map();
     this.__saveInstanceProperties();
     // ensures first update will be caught by an early access of
     // `updateComplete`
     this.requestUpdate();
-    (this.constructor as typeof ReactiveElement)._initializers?.forEach((i) =>
+    (this.constructor as typeof ReactiveElement).__initializers?.forEach((i) =>
```

Comment:
I think at one point we wanted to have a protected entry point to override that supported doing stuff before/after the super call (which you can't do in the constructor). 

This change seems fine, but is there a specific reason to make the change? Just YAGNI?

Reply:
Talked offline, and this was prompted by a user accidentally overriding `_initializer`.